features/svg-opacity.yml

```
@@ -0,0 +1,9 @@
+name: SVG opacity
```

Let's rename this file to `opacity-svg.yml` and change the title like so:

We don't have a lot of cases where this happens, but we kinda-sorta have a practice of using parentheses for disambiguation text. See https://github.com/web-platform-dx/web-features/pull/1492 for another example

```suggestion
name: Opacity (SVG)
```